src/components/Form.tsx

```diff
@@ -119,21 +127,15 @@ export class UnwrappedForm extends Component<IFormWrappedProps> {
   }
 }
 
-// istanbul ignore next
-const WrappedForm = AntForm.create()(UnwrappedForm);
-
-@autoBindMethods
-@observer
-export class Form extends Component<IFormProps> {
-  public static defaultProps = {
-    ...formPropsDefaults,
-    ...sharedComponentPropsDefaults,
-    showControls: true,
-  };
-
-  public render() {
-    return <WrappedForm {...this.props} />;
-  }
+function Form(props: IFormProps) {
+  const [form] = AntForm.useForm();
+  return <UnwrappedForm {...props} form={form} />;
 }
 
+Form.defaultProps = {
```

Comment:
we usually define `const defaultProps` at the top of the file. if we want to maintain consistency